github.js

```diff
@@ -67,6 +67,7 @@ var GithubLocation = function(options, ui) {
     throw 'Git not installed. You can install git from `http://git-scm.com/downloads`.';
   }
 
+  this.strictSSL = options.strictSSL;
```

Comment:
You're right, I missed that one.  Sorry about that.  I have submitted a fix.  Thanks for your patience!